diff --git a/dist/cli/build/bundle-server.js b/dist/cli/build/bundle-server.js
index 59f84c6..bf3e0da 100644
--- a/dist/cli/build/bundle-server.js
+++ b/dist/cli/build/bundle-server.js
@@ -103,5 +103,6 @@ export async function bundleServer(buildOpts, projectOpts) {
         external: [
             "./middleware/handler.mjs",
+            "cloudflare:sockets",
             // Do not bundle og when it is not used
             ...(useOg ? [] : ["next/dist/compiled/@vercel/og/index.edge.js"]),
         ],
